backend/src/components/user.js

```diff
@@ -158,17 +158,11 @@ async function getUserInfo(req, res) {
       application.facilityList = parseFacilityData(ap, userResponse.facilities);
 
       //add in funding agreement details based on the fiscal year
-      let fundingAgreementForFront = null;
-      for (const fundingAgreementObj of fundingAgreementDetails) {
-        if (fundingAgreementObj._ccof_programyear_value != application.ccofProgramYearId) {
-          continue;
-        } else if (!fundingAgreementForFront || fundingAgreementObj.ccof_version > fundingAgreementForFront.ccof_version) {
-          fundingAgreementForFront = fundingAgreementObj;
-        }
+      let fundingAgreementForFront = fundingAgreementDetails.find((fa) => fa._ccof_programyear_value === application.ccofProgramYearId && fa.statuscode === 1);
+      if (fundingAgreementForFront) {
+        fundingAgreementForFront = new MappableObjectForFront(fundingAgreementForFront, FundingAgreementMappings).data;
```

Comment:
I know this is older code but it's confusing to reuse the same variable for what are two different types. Can we just call the first object fundingAgreement or activeFundingAgreement?